app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2 class="text-2xl text-blue-500 mb-16">Sign up</h2>
```

Comment:
# [OPTIONAL]
- Here, the design looks different from the one provided to you. ou added a red colour and did not align the input fields as seen in the original design

### Your design
![image](https://github.com/shihabrafio/Budget_App/assets/45460192/43fe66e8-1d5b-4c1d-96ce-e804254928ec)


### Design Guidelines
![image](https://github.com/shihabrafio/Budget_App/assets/45460192/1e589052-0b50-4477-a696-7fd5f0e9eed1)